src/dotnet-aspnet-codegenerator/DotNetBuildCommandHelper.cs

```diff
@@ -54,6 +54,21 @@ internal class DotNetBuildCommandHelper
                 StdOut = stdOutMsgs
             };
         }
+
+        private static string GetFrameworkShortName(NuGetFramework framework)
+        {
+            string shortFramework = string.Empty;
+            if (framework != null)
+            {
+                if (framework.Framework.Equals(FrameworkConstants.FrameworkIdentifiers.Net) &&
```

Comment:
You should be checking for the frameworkidentifier

Reply:
framework.Framework is the framework identifier(".NETFramework"/ ".NETCore").